### ISSUE TITLE
[Bug] (assets): Color contrast requirements are not met for Colored icons in dark mode

### ISSUE THREAD

**Issue by ValentinaKozlova** [microsoft/fluentui-system-icons#977]
### Package version

2.0.x

### Current Behavior

Some colored icons in dark mode are not accessible. Color contrast requirements are not met.
Some parts have good contrast but some are less that 3:1. According to WCAG ratio should be 3:1 but we have small colorful elements. Probably for them it would be nice to have at least 4:1. 
In case with dark mode with background `#000` or `121212` majority of icons meet contrast requirements.
But in case with lighter background (e.g. `222`) some of them are not.

<img width="89" height="56" alt="Image" src="https://github.com/user-attachments/assets/b1457128-430b-4f36-9e32-709f723fc823" />
<img width="93" height="60" alt="Image" src="https://github.com/user-attachments/assets/2c7de6ab-0af3-490c-8e85-42c6b4c0162a" />
<img width="73" height="54" alt="Image" src="https://github.com/user-attachments/assets/26755a67-4ed0-4538-b328-ac1abff24c3e" />

**Note:** Related bug to colored icons #951 

### Expected Behavior

- Meaningful graphical icons must meet a minimum contrast ratio of 3:1 between the icon (foreground) and its adjacent background (non-text contrast, WCAG 2.1 SC 1.4.11).
- If the icon contains text or is functioning as text (e.g., letter, number, or word inside the icon), apply text contrast: 4.5:1 for normal text, 3:1 for large text (WCAG 2.0 SC 1.4.3).
- Decorative icons that convey no information do not require contrast compliance.

### Reproduction
see image attached + steps to repro

### Steps to reproduce

1. Go to https://storybooks.fluentui.dev/react/?path=/docs/icons-catalog--docs
2. Filter "Color"
3. Enable Dark mode.

### Are you reporting an Accessibility issue?

yes

### Suggested severity

Low - Has or doesn't need a workaround

### Are you willing to submit a PR to fix?

yes

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.
